nature_of_code/chp_03_oscillation/3_01_angular_motion.rs

```diff
@@ -11,49 +12,54 @@ fn main() {
 
 struct Model {
     angle: f32,
-    a_velocity: f32,
-    a_acceleration: f32,
+    angle_velocity: f32,
+    angle_acceleration: f32,
 }
 
 fn model(app: &App) -> Model {
     let angle = 0.0;
-    let a_velocity = 0.0;
-    let a_acceleration = 0.0001;
+    let angle_velocity = 0.0;
+    let angle_acceleration = -0.0001;
     app.new_window().size(800, 200).view(view).build().unwrap();
     Model {
         angle,
-        a_velocity,
-        a_acceleration,
+        angle_velocity,
+        angle_acceleration,
     }
 }
 
 fn update(_app: &App, model: &mut Model, _update: Update) {
-    model.angle += model.a_velocity;
-    model.a_velocity += model.a_acceleration;
+    model.angle += model.angle_velocity;
+    model.angle_velocity += model.angle_acceleration;
 }
 
 fn view(app: &App, model: &Model, frame: Frame) {
     // Begin drawing
     let draw = app.draw();
-    draw.background().color(WHITE);
+    draw.rect()
+        .wh(app.window_rect().wh())
+        .color(rgba(1.0, 1.0, 1.0, 0.05));
 
     draw.line()
         .start(pt2(-60.0, 0.0))
         .end(pt2(60.0, 0.0))
-        .color(BLACK)
+        .color(rgb8(0, 0, 0))
+        .stroke_weight(2.0)
         .rotate(model.angle);
 
     draw.ellipse()
-        .x_y(60.0, 0.0)
+        .xy(pt2(60.0, 0.0).rotate(model.angle))
         .w_h(16.0, 16.0)
-        .color(BLACK)
-        .rotate(model.angle);
+        .gray(0.5)
+        .stroke_weight(2.0)
+        .stroke_color(rgb8(0, 0, 0));
 
     draw.ellipse()
-        .x_y(-60.0, 0.0)
+        .xy(pt2(-60.0, 0.0).rotate(model.angle))
```

Comment:
For clarity, could the `.xy` and `.rotate` be on separate lines?

Reply:
running `cargo fmt` will dictate how this is laid out. Although I agree in this instance it might be more clear to have them on separate lines I'm still in favor of letting `cargo fmt` have control over all styling and formatting in nannou.

Reply:
I have `cargo fmt` configured to format everything on save, I will check again it's options

Reply:
Good point about using `cargo fmt`